src/iop/watermark.c

```diff
@@ -128,9 +140,13 @@ int legacy_params(dt_iop_module_t *self, const void *const old_params, const int
     n->rotate = 0.0;
     n->sizeto = DT_SCALE_IMAGE;
     g_strlcpy(n->filename, o->filename, sizeof(n->filename));
+    g_strlcpy(n->text, _("text"), sizeof(n->text));
+    g_strlcpy(n->font, "DejaVu Sans 10", sizeof(n->font));
+    n->color[0] = n->color[1] = n->color[2] = 127;
```

Comment:
Later on you use GdkRGBA.
From https://developer.gnome.org/gdk3/stable/gdk3-RGBA-Colors.html:
```
All values are in the range from 0.0 to 1.0 inclusive. So the color (0.0, 0.0, 0.0, 0.0) represents transparent black and (1.0, 1.0, 1.0, 1.0) is opaque white. Other values will be clamped to this range when drawing.
```

So there is no way you should be specifying 127 here and in other places...